pandas/core/frame.py

```diff
@@ -1697,19 +1697,45 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
 
         .. versionadded:: 0.21.0
 
+        This function writes the dataframe as a parquet file (see
+        :func:`pandas.io.parquet.to_parquet`). You can choose different parquet
+        backends, and have the option of compressing.
+
         Parameters
         ----------
         fname : str
-            string file path
+            String file path.
         engine : {'auto', 'pyarrow', 'fastparquet'}, default 'auto'
             Parquet library to use. If 'auto', then the option
             ``io.parquet.engine`` is used. The default ``io.parquet.engine``
             behavior is to try 'pyarrow', falling back to 'fastparquet' if
             'pyarrow' is unavailable.
         compression : {'snappy', 'gzip', 'brotli', None}, default 'snappy'
             Name of the compression to use. Use ``None`` for no compression.
-        kwargs
-            Additional keyword arguments passed to the engine
+        kwargs : dict
+            Additional keyword arguments passed to the parquet library. See
+            the documentation for :func:`pandas.io.parquet.to_parquet` for
+            complete details.
+
+        Returns
+        -------
+        None
+
+        See Also
+        --------
+        read_parquet : Read a parquet file.
+        DataFrame.to_csv : Write a csv file.
+        DataFrame.to_sql : Write to a sql table.
+        DataFrame.to_hdf : Write to hdf.
+
+        Notes
+        -----
+        This function requires either the fastparquet or pyarrow library.
```

Comment:
Can you make fastparquet and pyarrow into links to their home page? 

The syntax is `` `pyarrow <https:// ...>`__ ``